dev-games/physfs/physfs-3.0.1-r1.ebuild

```diff
@@ -1,12 +1,12 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
Why, though? 

EAPI-6 is still perfectly valid. Anyway my point is EAPI-bump should come with a revbump.

Reply:
Regardless of the EAPI change, this should have been done with a revbump because the patch fixes a runtime issue, not a build time issue.

Reply:
Yes, thank you! Sorry for the silly mistake:(